test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -2080,6 +2193,352 @@ def test_cgroup_use_hierarchy(self):
         else:
             self.assertFalse(1, "File %s not present" % fpath)
 
+    def check_req_rjs(self):
+        """
+        Check the requirements for the reliable job startup tests.
+        Return 1 if requirements are not satisfied.
+        """
+        ret = 0
```

Comment:
You don't use ret until the very end of the method. It's not needed since you can just return zero or one at that point.

Reply:
Done.